modules/pushapk_scriptworker/manifests/settings.pp

```diff
@@ -219,6 +219,13 @@
                             'google_credentials_file' => "${root}/fenix_beta.p12",
                             'certificate_alias' => 'fenix-beta',
                         },
+                        'production' => {
```

Comment:
Same question as above, any reason why we're not opting for `fenix-release` here?

Reply:
You mentioned in slack if "production" made sense as the name of the "main" channel.
I've found that there's a general convention to name the product's "main" channel as "production". Usually your other releases will have names like "nightly", "staging", "beta", "qa", and more, but the primary environment/channel tends to be "production".

If your product only has one release channel (there's no beta/staging/etc), then sometimes it will just be called "release".

This lines up with `gradle` configuration, which expects only a single release channel by default, so calls it `release` (e.g.: `assembleRelease`). Now that we've configured `gradle` for each of our three channels (nightly/beta/production), the term "release" is too general to refer to our "main" release.

I hope this makes sense, let me know if you want to chat :)